### ISSUE TITLE
[torrentleech-pl] (updating) Login failed: Selector "a[href^="logout.php"]" didn't match

### ISSUE THREAD

**Issue by kramarz89** [Jackett/Jackett#14217]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using FlareSolverr: No
Using proxy: Disabled
App config/log directory: /config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: Yes (image build: v0.20.3767-ls100)
File /etc/issue: Welcome to Alpine Linux 3.17
Jackett variant: CoreLinuxMuslAmdx64
OS version: Unix 5.10.0.0 (64bit OS) (64bit process)
Environment version: 6.0.15 (/app/Jackett/)
Starting Jackett v0.20.3767
```

### Description

As in the tittle, It wont work for me, I dont have the access blocked to the website, I've tried deleting the cookie. 

### Logged Error Messages

```text
2023-04-03 13:02:06	Error	
Jackett.Common.ExceptionWithConfigData: Login failed: Selector "a[href^="logout.php"]" didn't match
```

**Comment by ilike2burnthing** [comment]
Read the troubleshooting steps here:
https://github.com/Jackett/Jackett/wiki/Troubleshooting/#your-cookie-did-not-work

Your cookie should look like this:
`cf_chl_prog=xxx; cf_clearance=xxxxx_xxxxx_xxxxx-nnnnn-n-nnn; PHPSESSID=xxxxx; uid=nnnnn; pass=xxxxx; hashv=xxxxx`
not this:
`cf_chl_prog%3Dxxx%3B%20cf_clearance%3Dxxxxx_xxxxx_xxxxx-nnnnn-n-nnn%3B%20PHPSESSID%3Dxxxxx%3B%20uid%3Dnnnnn%3B%20pass%3Dxxxxx%3B%20hashv%3Dxxxxx`

**Comment by kramarz89** [comment]
Hi @ilike2burnthing, The cookie looks correct: (i've cut out some parts of it to obfuscate)

cf_clearance=0RdCssxxxxxxxODVogUM.zFxxxxx6cgvkjt38-168xxxxx693-0-250; PHPSESSID=9mgkxxxxxx79kk7xxxxxx9l7; uid=25xxx01; pass=ec6exxxxxxxxxxxxxxxx001853c75; hashv=977d9exxxxxxxxxxxxx650ec9

**Comment by ilike2burnthing** [comment]
Have you followed the troubleshooting steps above?

**Comment by kramarz89** [comment]
Ok, I think We can close this, turns out that torrentleech.pl doesnt allow for setting up Jackett, blocks all VPN's and what not, So I cannot access the site from the same server that Jackett resides on. Shame.

**Comment by ilike2burnthing** [comment]
Jackett is working fine for me with torrentleech.pl, though their FAQ does explicitly forbid the use of proxies, Tor, or VPNs to access the site.

If you can't access the tracker in browser from the same IP as Jackett without a proxy or VPN then you can't use the tracker at all, with or without Jackett.

Look at changing your DNS if you haven't already.

**Comment by kramarz89** [comment]
@ilike2burnthing 
Ive messaged the admin, he added the server IP to tracker whitelist, however, he said that Sonarr and Jackett are blocked globally.

**Comment by ilike2burnthing** [comment]
It's not blocked, I can use it without issue. Do they mean Jackett is banned? If so, we'd have to remove the indexer.

**Comment by kramarz89** [comment]
I mean, I wrote to the admin on the site, he told me that both are
"blocked" but I don't know what he means by that.. Can he actually
check/see/notice I would be using Jackett, lets say with sonarr for
example? Is it noticeable?

pon., 3 kwi 2023 o 23:30 ilike2burnthing ***@***.***>
napisał(a):

**Comment by ilike2burnthing** [comment]
K, well as it's unclear we'll leave it for now, they can contact up pretty easily and haven't.

I've had an account for ~2 years and haven't had any problems. While they could see and block our user-agent, they haven't.

**Comment by kramarz89** [comment]
Hi @ilike2burnthing I had some discussions with the admins, and they asked me what user-agent is jackett using, and i took one from other trackers that i could add as indexer. And after that I got a following reply: 
(translated from polish):

User-Agent ok. The problem is that the IP is hitting the path / and this address is globally blocked. I allowed him the address only for tracker and RSS, so any other URI will be dropped. Secondly, our RSS works in two ways: cookies - to display news in the RSS reader and passkey for automatic download. Unfortunately, restrictions on IPs are put in place for security reasons and possible abuse. I see that Ultra.cc has decided to use DataCamp Limited/CDN77, and there is a lot of crap there, like VPNs, proxies, etc.
To recap: only RSS with passkey loaded directly into the client will work for him.
He can generate a link for himself here:
https://torrentleech.pl/getrss.php

----------------
Is it something I can work with?
I've tried accessing the website using a VPN that apparently has the same IP as Jackett, but the IP is banned, So i guess there's not much else I can do, If I cannot get hold of the cookie from the same IP as Jackett?

**Comment by ilike2burnthing** [comment]
Changed to credentials login rather than cookies, users will now need to install and configure FlareSolverr, if they haven't already - https://github.com/Jackett/Jackett#configuring-flaresolverr. Note that Jackett and FlareSolverr must use the same public IP (if one uses a VPN, the other needs to use the same server).

However, if the IP your VPS/seedbox is using is completely blocked, this won't help (nor would having a cookie from the IP). You'd need to use an unblocked VPN on your VPS/seedbox. If this isn't an option, then no there's nothing you can really do other than change provider.

Also, `/` was being used as the login test path. While I wasn't having issues with it, I've changed to to `index.php`.

**Comment by kramarz89** [comment]
Alright, cheers for the willingness and help, But as you've said, guess I need to drop the idea of using Jackett in case of this site, and just continue manually, because of the protections in place. 
Thanks!
